node/src/BaseClient.ts

```diff
@@ -3235,6 +3237,34 @@ export class BaseClient {
         return this.createWritePromise(createZPopMin(key, count));
     }
 
+    /**
+     * Blocks the connection until it removes and returns a member with the lowest score from the
+     * first non-empty sorted set, with the given `key` being checked in the order they
+     * are provided.
+     * `BZPOPMIN` is the blocking variant of {@link zpopmin}.
+     *
+     * See https://valkey.io/commands/bzpopmin/ for more details.
+     *
+     * @remarks When in cluster mode, `keys` must map to the same hash slot.
+     * @param keys - The keys of the sorted sets.
+     * @param timeout - The number of seconds to wait for a blocking operation to complete. A value of
+     *     `0` will block indefinitely. Since 7.0.0: timeout is interpreted as a double instead of an integer.
```

Comment:
> Since 7.0.0: timeout is interpreted as a double instead of an integer.
Does this matter here? It's always a `number` in the client.

Reply:
I think it's still relevant since this basically informs the user that its possible to pass in decimal timeout values and it won't be round up post 7.0 and vice versa